packages/react-components/source/react/library/breadcrumb/Breadcrumb.js

```diff
@@ -28,13 +36,33 @@ const Breadcrumb = ({ children, className, ...props }) => {
   });
 
   return (
-    <nav
-      aria-label="Breadcrumb"
-      className={classNames('rc-breadcrumb', className)}
-      {...props}
-    >
-      <ol>{crumbs}</ol>
-    </nav>
+    <>
+      {type === 'standard' ? (
+        <nav
+          aria-label="Breadcrumb"
+          className={classNames('rc-breadcrumb', className)}
+          {...props}
+        >
+          <ol>{crumbs}</ol>
+        </nav>
+      ) : (
+        <div
+          className={classNames('rc-breadcrumb', className)}
+          aria-label="Breadcrumb"
+        >
+          <Icon type="chevron-left" aria-hidden="true" />
+          <Hyperlink
+            className="rc-breadcrumb-section"
+            color="medium"
+            size="tiny"
+            tabIndex="0"
+            {...props}
+          >
+            {backLabel}
+          </Hyperlink>
```

Comment:
👍 I tested that this works with with `as` pattern since you're spreading props onto the Link here.